sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs

```diff
@@ -742,10 +627,14 @@ public virtual async Task<ServiceBusReceivedMessage> PeekAsync(CancellationToken
             var lockTokenGuids = receivedMessages.Select(m => new Guid(m.LockToken)).ToArray();
             if (lockTokenGuids.Any(lockToken => RequestResponseLockedMessages.Contains(lockToken)))
             {
-                string receiveLinkName = Consumer.GetReceiveLinkName();
-                return Connection.DisposeMessageRequestResponseAsync(lockTokenGuids, timeout, DispositionStatus.Completed, IsSessionReceiver, Session.UserSpecifiedSessionId, receiveLinkName);
+                return _innerReceiver.DisposeMessageRequestResponseAsync(
+                    lockTokenGuids,
+                    timeout,
+                    DispositionStatus.Completed,
+                    IsSessionReceiver,
+                    SessionManager.SessionId);
```

Comment:
If we decide to use the `AmqpReceiver` class properties then we can remove IsSessionReceiver and sessionId.

Reply:
YEs